### PR TITLE
Csv Output

### PR DESCRIPTION
**Pull request by cacraig**
- Outputs CSV with the &output=csv option on the /incidents, and /incidents/count endpoints.

Addresses: #27

### CODE REVIEW

**Comment by cacraig** [comment]
Hey, @catherinedevlin or @harrisj. When you get a chance, can you take a look at this? Thanks!

File: crime_data/common/base.py [hunk 1]
```diff
@@ -141,48 +141,52 @@ class CdeResource(Resource):
                  '<': '__le__',
                  '==': '__eq__', }
 
+
+    def _serialize_dict(self, data, accumulator={}):
```
Comment (cacraig) [inline crime_data/common/base.py]: It's probably going to change a bit, depending on testing feedback. At some point we'll flesh out unit tests for the common.base module.

**Comment by harrisj** [comment]
A few notes on the output (generally pretty awesome!)

`incidents.csv`
Looks like the comment field is being concatenated without spaces, eg `CC:  Ms. Amber LozanoSupervisorCriminal Justice Statistics CenterDepartment of Justice`

`incident_counts.csv`
Do you think we could make the year the first column?

**Comment by cacraig** [comment]
@harrisj  - It's not super trivial to sort the columns, but I'm going to get started on that after this is pushed. -> https://github.com/18F/crime-data-api/issues/80 . 
  
Regarding the comment field - > There's no concatenation that is going on in the code, the the value should be exactly what is in the Database.

**Comment by harrisj** [comment]
Ooof, I hate it when the source data is like that. Thanks! LGTM

**Pull request merged**